krypto-api/src/main/java/com/kryptnostic/krypto/engine/KryptnosticEngine.java

```diff
@@ -137,31 +133,29 @@ protected void finalize() throws Throwable {
      * NEVER CALL THIS FROM THE SERVER
      **/
     /*
-     * Function: getObjectSearchPairFromObjectIndexPair(objectIndexPair)
-     * Returns a serialized pair of (FHE-encrypted ObjectSearchKey, ObjectAddressMatrix)
+     * Function: getObjectSearchPairFromObjectIndexPair(objectIndexPair) Returns a serialized pair of (FHE-encrypted
+     * ObjectSearchKey, ObjectAddressMatrix)
      */
-    public native byte[] getObjectSearchPairFromObjectIndexPair(byte[] objectIndexPair);
+    public native byte[] getObjectSearchPairFromObjectIndexPair( byte[] objectIndexPair );
 
     /**
      * NEVER CALL THIS FROM THE SERVER
      **/
     /*
-     * Function: getObjectSearchPairFromObjectSharePair(objectSearchPair)
-     * Returns a serialized pair of (FHE-encrypted objectSearchKey, objectConversionMatrix)
-     * Sent by a client to another to share a document
+     * Function: getObjectSearchPairFromObjectSharePair(objectSearchPair) Returns a serialized pair of (FHE-encrypted
+     * objectSearchKey, objectConversionMatrix) Sent by a client to another to share a document
```

Comment:
These mostly look like cases where we should be updating the formatter to not format these comments